lib/natalie/compiler/binding_gen.rb

```diff
@@ -603,7 +603,6 @@ def generate_name
 
 gen.binding('Range', 'initialize', 'RangeValue', 'initialize', argc: 2..3, pass_env: true, pass_block: false, return_type: :Value)
 gen.binding('Range', 'begin', 'RangeValue', 'begin', argc: 0, pass_env: false, pass_block: false, return_type: :Value)
-gen.binding('Range', 'first', 'RangeValue', 'begin', argc: 0, pass_env: false, pass_block: false, return_type: :Value)
```

Comment:
Actually, Range has its own `first` method:

```ruby
irb(main):001:0> r = 1..10
=> 1..10
irb(main):002:0> r.method(:first)
=> #<Method: Range#first(*)>
irb(main):003:0> r.method(:first).owner
=> Range
```

Reply:
Mh yeah.. I'll look into a possible alternative implementation for Range#first

Reply:
Added a new implementation for Range#first that supports an integer argument.